Exiled.API/Features/Door.cs

```diff
@@ -516,11 +549,24 @@ public void Lock(float time, DoorLockType lockType)
         /// <param name="flagsToUnlock">The <see cref="Enums.DoorLockType"/> of the lockdown.</param>
         public void Unlock(float time, DoorLockType flagsToUnlock) => DoorScheduledUnlocker.UnlockLater(Base, time, (DoorLockReason)flagsToUnlock);
 
+        /// <summary>
+        /// Checks if specified <see cref="Player"/> can interact with the door.
+        /// </summary>
+        /// <param name="player">Player to check.</param>
+        /// <returns><see langword="true"/> if anybody can interact. Otherwise, false.</returns>
+        public bool IsAllowToInteract(Player player) => Base.AllowInteracting(player.ReferenceHub, 0);
+
+        /// <summary>
+        /// Checks if anybody can interact with the door.
+        /// </summary>
+        /// <returns><see langword="true"/> if anybody can interact. Otherwise, false.</returns>
+        public bool IsAllowToInteract() => Base.AllowInteracting(null, 0);
+
```

Comment:
```suggestion
        /// <summary>
        /// Checks if specified <see cref="Player"/> can interact with the door.
        /// </summary>
        /// <param name="player">Player to check.</param>
        /// <returns><see langword="true"/> if anybody can interact. Otherwise, false.</returns>
        public bool IsAllowToInteract(Player player = null) => Base.AllowInteracting(player?.ReferenceHub, 0);

```